modules/user-groups/user-groups.php

```diff
@@ -687,9 +687,13 @@ function user_profile_update( $errors, $update, $user ) {
 		if ( !$update )
 			return array( &$errors, $update, &$user );
 
-		//Don't allow update of user groups from network
-		if ( ( !is_null( get_current_screen() ) ) && ( get_current_screen()->is_network ) )
-			return;
+		// `get_current_screen()` is defined on most admin pages, but not all.
+		if( function_exists( 'get_current_screen' ) ){
+			//Don't allow update of user groups from network
+			if ( ( !is_null( get_current_screen() ) ) && ( get_current_screen()->is_network ) ) {
```

Comment:
Minor suggestion to fix the double function call and improve readability a bit:

```suggestion
			$screen = get_current_screen();
			if ( ! is_null( $screen ) && $screen->is_network ) {
```